### PR TITLE
Update service proxy agent image version

### PR DESCRIPTION
**Pull request by jasonwzm**

### CODE REVIEW

**Comment by jasonwzm** [comment]
Please hold off from merging for now.

File: scripts/asm-installer/asm_vm [hunk 1]
```diff
@@ -33,7 +33,7 @@ readonly ASM_META_VERSION
 readonly ASM_REVISION_PREFIX
 RELEASE=""
 
-SERVICE_PROXY_AGENT_BUCKET="${SERVICE_PROXY_AGENT_BUCKET:=gs://gce-service-proxy/service-proxy-agent/releases/service-proxy-agent-asm-${ASM_META_VERSION}-stable.tgz}"
```
Comment (zerobfd) [inline scripts/asm-installer/asm_vm:36]: Is ASM_META_VERSION used anywhere else?

**Review by zerobfd — APPROVED** [review]
lgtm, feel free to merge whenever you want

File: scripts/asm-installer/asm_vm (hunk 1 above)
Comment (jasonwzm) [reply scripts/asm-installer/asm_vm:36]: Yes, https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/3d1fdddf81f1dbec5e94ab7b023ea112f59e67f9/scripts/asm-installer/asm_vm#L1249

**Pull request merged**